src/components/utils/Svg.css

```diff
@@ -0,0 +1,53 @@
+.worker,
+.folder {
+    position: relative;
+    top: 2px;
+}
+
+.arrow,
+.folder,
+.domain,
+.file,
+.worker,
+.refresh,
+.add-button {
+    fill: var(--theme-splitter-color);
+}
+
+.domain,
+.file,
+.worker,
+.refresh,
+.add-button {
+    position: relative;
+    top: 1px;
+}
+
+.domain svg,
+.folder svg,
+.worker svg,
+.refresh svg,
+.add-button svg {
+    width: 15px;
+    fill: currentColor;
```

Comment:
It was a mistake. It is [fixed](https://github.com/andreicristianpetcu/debugger.html/commit/4cabd2e528c09c839b8cd0ff8c381910fc4d174c) [now](https://github.com/andreicristianpetcu/debugger.html/commit/2ba1c7d9acde2d088131324b79120b06de59cef8). I wanted to make the refresh and + buttons have a single color and not change on :hover but I made the selectors a little bit too generic.